src/javascript/app/pages/user/account/statement/statement.init.js

```diff
@@ -51,6 +52,15 @@ const StatementInit = (() => {
         return chunk;
     };
 
+    const getAccountStatistics = () => {
+        // only show Account Statistics to MLT/MX clients
+        if (!/^MLT|MX/.test(Client.get('loginid'))) return;
```

Comment:
please check using the landing company shortcode, the login id can change so it should never be relied upon unless there's absolutely no choice

Reply:
okay sure